test/webpack-build.test.js

```diff
@@ -238,6 +238,24 @@ for (const nodeVersion of SUPPORTED_NODE_VERSIONS) {
     test.false(result.hasErrors());
   });
 
+  test.serial(`Can webpack files that use arrow functions inside async functions when targetting ${nodeVersion}`, async (test) => {
+    const source = path.join(__dirname, 'fixtures', 'async_with_arrow.js');
+
+    const result = await build({
+      nodeVersion,
+      entrypoint: source,
+      outputPath: test.context.buildDirectory,
+      serviceName: 'test-service'
+    });
+    test.false(result.hasErrors());
+
+    // Try loading the newly loaded file to make sure it can
+    // execute without an error
+
+    // eslint-disable-next-line security/detect-non-literal-require
+    require(path.join(test.context.buildDirectory, 'async_with_arrow'));
```

Comment:
What is the rationale for loading the module directly rather than invoking it via the Lambda runtime? It seems like that would avoid the need for the environment variable mangling in the fixture and would be a more detailed test as it would ensure that the bundled code is compatible with the Node version in the node runtime (as opposed to whatever is on the caller's system).

Reply:
I didn't think of loading with the lambda runtime. Good idea. I can fix that after dinner

Reply:
Your idea sounded good, but when I looked into it I didn't find a method like `lambda.useLambda(test, options)` that would just let me run a lambda execution given an a module. Do you think such a thing already exists or were you thinking that I would refactor to allow an easy single execution?

Reply:
I was thinking more along the lines of restructuring the test. The fixture is only marginally different than a lambda module.

Reply:
Was looking at this some more. I typically split these sorts of tests out into separate files so that I can use `lambda.useLambda()` as you suggest. However, the `LambdaRunner` is exported and there is a `createContainer()` helper. Using those two together should make it pretty straight forward to setup a bundled lambda on demand (rather than relying on test seutp). A new helper like that would be pretty nice for better verification in other webpack tests.